File: 2025_03/leetcode2025-03-15.py
class Solution:
    def scoreOfString(self, s: str) -> int:
        res = 0
        n = len(s)
        for i in range(1,n):
            res += abs(ord(s[i])-ord(s[i-1]))
        return res

if __name__ == "__main__":
    sl = Solution()
    s = "hello"
    print(sl.scoreOfString(s))

File: 2025_03/leetcode2025-03-15.cpp
#include <iostream>
#include <string>
using namespace std;

class Solution {
public:
    int scoreOfString(string s) {
        int res = 0;
        int n = s.size();
        for (int i=1; i<n; i++){
            res += abs(s[i]-s[i-1]);
        }
        return res;
    }
};

int main(){
    Solution sl;
    string s = "hello";
    cout << sl.scoreOfString(s) << endl;
}
